release-notes.hbs.md

```diff
@@ -332,6 +332,11 @@ This release has the following known issues, listed by area and component.
   workload isn't running. For more information, see
   [Troubleshooting](vs-extension/troubleshooting.hbs.md#del-wrkld-not-running).
 
+#### <a id="1-4-0-vs-app-acc"></a> Tanzu Application Accelerator for Visual Studio
```

Comment:
This should be updated from `Visual Studio` to `VS Code`

Reply:
updated in the latest commit